crypto/keystack.ts

```diff
@@ -1,7 +0,0 @@
-// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
-// This module is browser compatible.
-
-/**
- * @deprecated (will be removed after 0.209.0) Import from `std/crypto/unstable/keystack.ts` instead.
- */
-export * from "./unstable/keystack.ts";
```

Comment:
Looks like we still need to keep this file

Reply:
Whoops. Fixed!